nexusutils/nexus/README.md

```diff
@@ -2,8 +2,20 @@ This utility outputs a debug log for a given NeXus file by annotating the data a
 metadata entries with the schema definitions from the respective NeXus base classes
 and application definitions to which the file refers to.
 
-```console
-user@box:~$ read_nexus <path_to_nexus_file>
+```
+Options:
+    --nexus-file : Name of nexus output file (.nxs) to enquiry.
```

Comment:
please make this option also available as -f  
and also take the argument with no option specification as the nexus-file. Documentation should read:
read_nexus [Options] <path_to_nexus_file>